.appveyor.yml

```diff
@@ -89,6 +89,10 @@ init:
   - git config --global core.symlinks true
 
 install:
+  # Because prebuilt/ is cached, we need to manually reset changes that might
+  # happen to the sha1 files here.
+  - git checkout -- prebuilt
```

Comment:
This was done previously in .travis.yml during the last sccache upgrade in #1576